test/testcafe/framework/page-objects/ChallengeOktaVerifyPushPageObject.js

```diff
@@ -46,4 +49,29 @@ export default class ChallengeOktaVerifyPushPageObject extends ChallengeFactorPa
     return this.form.getElement(FORM_INFOBOX_WARNING);
   }
 
+  subtitleExists() {
```

Comment:
nit: `hasSubtitle()` (and we can probably add this to the parent PageObject, not sure if any other test needs it, but seems that it is a good candidate to be reused

Reply:
have kept the name as `subtitleExists` as thats being followed for similar 'exist' check. and moved it to `BasePageObject`